tutorials/intellijDebugger.md

```diff
@@ -6,7 +6,140 @@ pageNav: 3
 
 # {{ title }}
 
+Before diving into tracing code with IntelliJ IDEA’s debugger, it’s essential to understand key debugger features and their purpose.
+
 This video (from LaunchCode) gives a pretty good explanation of how to use the IntelliJ IDEA debugger.
 
 @[youtube](1bCgzjatcr4)
 
+## Starting A Debugging Session
+
+To utilise the debugger, we need to make sure that the code is running in debug mode.
+
+### Run Debug Mode
```

Comment:
Logically, setting a break point should come first. That's why it comes first in the video as well.